src/makefile

```diff
@@ -5,21 +5,19 @@ include prorab-license.mk
 
 # ===============================
 # ========= model_obj ===========
```

Comment:
remove the whole commented `model_obj` section

Reply:
removed